backend/src/main/java/com/empire/World.java

```diff
@@ -398,33 +398,28 @@ public static World startNew(String gmPasswordHash, String obsPasswordHash, Map<
 				if (i == 0) {
 					c.addTag("Ruler");
 					if ("checked".equals(setup.food)) c.values.add("food");
-					if ("checked".equals(setup.prosperity)) c.values.add("prosperity");
 					if ("checked".equals(setup.happiness)) c.values.add("happiness");
 					if ("checked".equals(setup.supremacy)) c.values.add("supremacy");
```

Comment:
Should this line also be removed now that supremacy has been nixed?

Reply:
Done. Also caught a bunch of other places, I think.